ios/FluentUI/People Picker/AvatarView.swift

```diff
@@ -250,16 +255,39 @@ open class AvatarView: UIView {
     /// Set this to override the avatar view's default accessibility label.
     @objc open var overrideAccessibilityLabel: String?
 
+    /// Used when avatarView doesn't have image or can't generate initials string
+    internal var preferredFallbackImageStyle: AvatarFallbackImageStyle = .outlined {
+        didSet {
+            if preferredFallbackImageStyle != oldValue {
+                if let fallbackImageStyle = fallbackImageStyle, fallbackImageStyle == oldValue {
+                    updateImageViewWithFallbackImage(style: preferredFallbackImageStyle)
+                }
+            }
+        }
+    }
+
     /// Initializes the avatar view with a size and an optional border
     ///
     /// - Parameters:
     ///   - avatarSize: The AvatarSize to configure the avatar view with
     ///   - hasBorder: Boolean describing whether or not to show a border around the avatarView
     ///   - style: The `MSAvatarStyle` to indicate whether the avatar should be displayed as a circle or a square
-    @objc public init(avatarSize: AvatarSize, withBorder hasBorder: Bool = false, style: AvatarStyle = .circle) {
+    @objc public convenience init(avatarSize: AvatarSize, withBorder hasBorder: Bool = false, style: AvatarStyle = .circle) {
+        self.init(avatarSize: avatarSize, withBorder: hasBorder, style: style, preferredFallbackImageStyle: .outlined)
+    }
+
+    /// Initializes the avatar view with a size and an optional border
+    ///
+    /// - Parameters:
+    ///   - avatarSize: The AvatarSize to configure the avatar view with
+    ///   - hasBorder: Boolean describing whether or not to show a border around the avatarView
+    ///   - style: The `MSAvatarStyle` to indicate whether the avatar should be displayed as a circle or a square
+    ///   - preferredFallbackImageStyle: `AvatarFallbackImageStyle` used when avatarView doesn't have an image or can't show initials text
+    @objc public init(avatarSize: AvatarSize, withBorder hasBorder: Bool = false, style: AvatarStyle = .circle, preferredFallbackImageStyle: AvatarFallbackImageStyle = .outlined) {
```

Comment:
Shouldn't preferredFallbackImageStyle be part of the setup methods? Or a public property?
Otherwise, it's impossible to change this value once the AvatarView has been created.

Reply:
didn't thought that client would need it but expose it now.